althea/althea-rust-binaries/Makefile

```diff
@@ -26,7 +26,7 @@ define Package/althea-rust-binaries
   TITLE:=Althea Rust executables
   URL:=https://github.com/althea-mesh/althea_rs
   MAINTAINER:= Justin Kilpatrick <justin@altheamesh.com>
-  DEPENDS:=@IPV6
+  DEPENDS:=@IPV6 +libopenssl +libsqlite3
```

Comment:
You want `@OPENSSL_WITH_COMPRESSION` instead of `+libopenssl`. The latter would include the default openssl config which will fail to supply the `openssl/comp.h` header. Does a vanilla build compile with those for you though?

Reply:
it compiles just fine in a clean environment. So one of us must be missing something.